portal/cookbook-guide.md

```diff
@@ -6,84 +6,256 @@ They are [geoscience](https://en.wikipedia.org/wiki/Earth_science)-focused
+If the build is successful, you should be able to preview the result in a web browser by running
+```
+open _build/html/index.html
+```
+
+### Strip output from your notebooks before committing
+
+Wherever feasible, we recommend only committing bare, unexecuted notebooks into your Cookbook repository. There are a few reasons:
+- The notebooks are executed automatically each time the book is built
+- Pre-existing output in the notebooks will have no effect on the published book
+- Better reproducibility of your content
+- The repo stays cleaner and leaner. Diffs of notebook changes are easier to interpret.
 
-- Add the notebooks to `_toc.yml` (the table of contents). See [`radar-cookbook/_toc.yml`](https://github.com/ProjectPythia/radar-cookbook/blob/main/_toc.yml) for syntax
-- Edit `README.md` as described in that file. This is the homepage of your cookbook, so it should be descriptive
-- If your cookbook requires more computing resources than available through GitHub Actions, change `execute_notebooks` from `cache` to `binder` in `_config.yml` to run your cookbook on the Pythia Binder
-- Clear all notebook outputs, since the Pythia infrastructure will execute the notebooks
-- Ensure that your cookbook successfully builds and shows the executed code
+To strip a notebook, in a Jupyter Lab session, go to "Kernel" &rarr; "Restart Kernel and Clear Output of All Cells...". Save your notebook, then commit.
 
-## Transfer cookbook to the [ProjectPythia](https://github.com/ProjectPythia) organization
+You're ready to push content up to GitHub and trigger the automated publishing pipeline.
 
-1. [Contact Project Pythia via the Pangeo Discourse](https://discourse.pangeo.io/c/education/project-pythia/60) (or otherwise) to let us know about your cookbook
-1. Someone from the Pythia team will add you as a member of the ProjectPythia organization
-1. Once you have accepted the invitation, navigate to the settings of your cookbook repository, scroll down to the Danger Zone, and click "Transfer"
-1. Select or type "ProjectPythia", confirm, and transfer
-1. Replace the `repository_url` in the `sphinx/config/html_theme_options` of the `_config.yml` file to point to your cookbook's new GitHub repository within the [ProjectPythia](https://github.com/ProjectPythia) organization
+## Deploying your Cookbook
 
-You will automatically retain write access to your cookbook, but if you would like to add outside collaborators to the repository, contact a member of the Pythia team to be given the Admin role on your cookbook repository.
+Pythia Cookbooks are not just collections of notebooks! They are backed by GitHub-based automation that handles building, testing, previewing, and publishing the books online.
 
-You can open issues, PRs, and continue making edits as necessary.
+### About the publishing pipeline
 
-## Make a Zenodo release of your Cookbook
+Any time you push new content to the `main` branch of your Cookbook repo, actions are triggered that do the following:
```

Comment:
So "your Cookbook repo" is a little misleading here? The actions are only triggered on the cookbook that is in the ProjectPythia organization, not your fork of the repo, correct?

Reply:
I added a note block at the bottom of [this section](https://projectpythia.org/_preview/435/cookbook-guide.html#where-is-my-book) to try to clarify this. 

We can use more precise language in the sentence you highlighted. I didn't want it to get too convoluted. Strictly speaking, it's the `main` branch of the `upstream` repository, not your personal fork. Unless you *haven't* transferred ownership of your repo, it which case it would be the `main` branch of your personal repository (which is not a fork and thus does not have an upstream equivalent).

Reply:
Any suggested language for this?

Reply:
Nope. That looks good.